sections/video.liquid

```diff
@@ -20,7 +20,7 @@
     <div{% if section.settings.full_width %} class="page-width"{% endif %}>
       {% unless section.settings.heading == "" %}
         <div class="title-wrapper title-wrapper--no-top-margin">
-          <h2 class="title">{{ section.settings.heading }}</h2>
+          <h2 class="title {{ section.settings.heading_size }}">{{ section.settings.heading }}</h2>
```

Comment:
here when the heading is empty we're still getting some margin from the element even though it's empty: 
![](https://screenshot.click/15-24-m5gds-sysi9.png)

Reply:
I cannot replicate this. If I remove text, the element is not outputted on the page for me. Can you try again and see if you are still seeing that Ludo?

Reply:
Im not sure why it happened in the first place. I can't replicate anymore 👍